docs/source/contributing/releasing.adoc

```diff
@@ -51,13 +51,25 @@ The automated release performs:
 If the CI release job is failing for a known reason then you can skip the integration tests by changing the `SKIP_INTEGRATION_TEST` value to `true` in Makefile.
 ====
 
+The systemtray bits needs to uploaded separately, which can be performed by triggering CI job as:
+
+----
+$ make ci_release_systemtray CIRCLECI_API_KEY=<api-key> RELEASE_VERSION=<version>
+----
+
+where
+
+  - `api-key` : {project} Circle CI API key
```

Comment:
@anjannath Just make sure you put that api-key to same release mojo page so that everyone use it instead creating their own.

Reply:
@praveenkumar Will do!